tests/pact/keysets.repl

```diff
@@ -125,3 +125,50 @@
 
 (env-keys ["c"])
 (enforce-keyset 'rotate-me)
+
+(begin-tx)
+
+
+;;; test caps
+
+(env-data { "a": ["a"], "ab": { "keys": ["a","b"], "pred": "keys-any" } })
+
+(module sig-caps G
+  (defcap G () true)
+
+  (defconst KEYSET_A (read-keyset "a"))
+  (defconst KEYSET_AB (read-keyset "ab"))
+
+  (defcap A (x:string y:bool)
```

Comment:
The variables `x` and `y` don't seem to be used in the `defcap`

Reply:
Caps function both as a parameterization/specification of the capability, as well as a function, the latter just being for running predicates to guard against improper granting.  Here the test is simply testing that the cap parameterization matches what was specified in the signature. I'll comment to clarify.